docker-hub/access-tokens.md

```diff
@@ -65,4 +68,23 @@ a password, enter your token instead.
 
 If you have 2FA enabled, you must use a personal access token when logging in
 from the Docker CLI. If you don't have it enabled, this is an optional (but
-more secure) method of authentication.
\ No newline at end of file
+more secure) method of authentication.
+
+## Modify existing tokens
+
+You can rename, activate, deactivate, or delete a token as needed.
+
+1. Access your tokens under **[Account Settings](https://hub-stage.docker.com/settings/general) > [Security](https://hub-stage.docker.com/settings/security)**.
+
+2. Select a token and click **Delete** or **Edit**, or use the menu on
+the far right of a token row to bring up the edit screen. You can also
+select multiple tokens to delete them all once.
+
+      ![Delete or Edit](images/hub-delete-edit-token.png){: {:width="700px"}}
+
+   > **Note**
+   >
+   > You can select multiple tokens and delete at onnce.
```

Comment:
typo: once

We've also mentioned about deleting the tokens in 2. Wondering if we should we repeat this here?

Reply:
Removed this sentence